inst/examples/02-components.Rmd

```diff
@@ -230,11 +230,21 @@ Theorems\index{theorem} and proofs are commonly used in articles and books in ma
 In **bookdown**, the types of theorem environments supported are in Table \@ref(tab:theorem-envs). To write a theorem, you can use the syntax below:
 
 ````markdown
-`r ''````{theorem}
-Here is my theorem.
-```
+::: theorem
+This is a `theorem` environment which can contain **any** _markdown_ syntax
```

Comment:
For code blocks, make sure the line width is less or equal to 69, otherwise it will exceed the page margin.

Reply:
Oh ok. That is specific. 69. I'll try to remember that. Thanks !

Reply:
It depends on the font size and typeface. 69 is for this book. In general, I'd try to stay within 60 characters if possible.